### DISCUSSION TITLE
'command not found: typer' in macOS

### DISCUSSION THREAD

**Discussion by ohadch** [tiangolo/typer-cli#146] [Questions] [answered]
Hey,

Trying to figure out what am I missing. I followed the instructions and installed the cli using `python -m pip install typer-cli`, but still getting `zsh: command not found: typer` error. 

I tried to check my `PATH`, `.zshrc` and `.bashrc`, nothing appears in either of them. What am I doing wrong? I'll be happy to provide any information required :)

Thanks!

**Comment by aanimesh23** [comment]
It is in your python `bin` folder.

for me it was at `/Users/apple/Library/Python/3.9/bin`
After identifying the location you can run `export PATH=$PATH:/Users/apple/Library/Python/3.9/bin` to solve the problem

**Comment by hswope-arcweb** [comment]
I am having the same issue. I looked in all the usual locations.

**Comment by fritz-astronomer** [comment]
`which python` or `where python` might help you

**Reply by tiangolo** [reply, marked as answer]
This is probably the best idea. :nerd_face: 

---

That will show you where you are installing `typer`, it should end up there in the same location. It could also be that you end up in some way installing it in a virtual environment but it is not "activated" yet (`source .venv/bin/activate`).

Now Typer-CLI is part of Typer, since version 0.12.1, so you can just install `pip install typer` and you get the `typer` command. :rocket: 

If you still have problems after trying with the latest version, please create a question discussion in the Typer repo: https://github.com/tiangolo/typer/discussions/new?category=questions
